pypy2/python.pypy.nuspec

```diff
@@ -3,24 +3,24 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>python.pypy</id>
-    <version>7.1.0</version>
+    <version>7.3.1</version>
     <packageSourceUrl>https://github.com/refi64/chocolatey-pypy</packageSourceUrl>
     <owners>Ryan Gonzalez</owners>
     <title>PyPy</title>
     <authors>The PyPy Developers</authors>
     <projectUrl>http://www.pypy.org</projectUrl>
-    <iconUrl>https://raw.githubusercontent.com/refi64/chocolatey-pypy/7.1.0/pypy-icon.png</iconUrl>
-    <copyright>2003-2019 The PyPy Developers</copyright>
-    <licenseUrl>https://bitbucket.org/pypy/pypy/src/release-pypy2.7-v7.1.0/LICENSE</licenseUrl>
+    <iconUrl>https://raw.githubusercontent.com/refi64/chocolatey-pypy/master/pypy-icon.png</iconUrl>
```

Comment:
This should be "pinned" to the tag we're going to introduce (`7.3.1`) otherwise the URL can bitrot as the `master` branch changes.

Reply:
Fixed.